apps/wallet/src/ui/app/pages/home/nfts/HiddenAsset.tsx

```diff
@@ -65,6 +66,28 @@ export default function HiddenAsset(item: HiddenAssetProps) {
             collectibleType: type!,
         });
     }
+
+    async function handleShowAsset() {
+        try {
+            await showAsset(objectId);
+            toast.success(
+                (t) => (
+                    <MovedAssetNotification
+                        t={t}
+                        destination="Visual Assets"
+                        onUndo={() => undoShowAsset(objectId)}
+                    />
+                ),
+                {
+                    duration: 4000,
```

Comment:
This was already like this before but, should we naybe move this to a  constant or nah? Just a thought, feel free to leave it like this, its really only used twice